apps/Dataman/src/screens/MaterialApproval.tsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import { View, Text, Button, WhiteSpace, List } from "@ant-design/react-native";
+import { StyleSheet } from "react-native";
+import { AntDesign } from "@expo/vector-icons";
+import { Navbar } from "../components/Navbar";
+import { PageHeader } from "../components/PageHeader";
+import { InputField } from "../components/InputField";
+
+const MaterialApproval = () => {
+  const Item = List.Item;
+
+  return (
+    <View>
+      <Navbar />
+      <PageHeader text="Material Request Approval" />
+      <List>
+        <Item arrow="horizontal">
+          <Item.Brief>
+            <Text style={styles.list_text}>Indent Date</Text>
+            <AntDesign name="questioncircleo" color="black" />
+          </Item.Brief>
+          <Text style={styles.text}>Enter Date</Text>
+        </Item>
+
+        <Item>
+          <Text style={styles.list_text}>Indent Number</Text>
+          <InputField placeholder="Enter Number" />
+        </Item>
+      </List>
+
+      <View style={styles.buttonContainer}>
+        <Button type="primary" style={styles.button}>
+          Submit
+        </Button>
+        <WhiteSpace size="lg" />
+        <Button type="warning" style={styles.button}>
+          Deny
+        </Button>
+      </View>
+    </View>
+  );
+};
+
+export default MaterialApproval;
+
+const styles = StyleSheet.create({
```

Comment:
why can't we use tailwind here

Reply:
If tailwind css is not working then  use ///reference or add a global environment file which declares tailwind css as it is mentioned in their documentation

Reply:
Tailwind does not work on the components of the ant design. If we want to use Tailwind, then we have to use React Native components. That's why I applied normal styling